torch/csrc/cuda/shared/nvtx.cpp

```diff
@@ -1,11 +1,41 @@
 #ifdef _WIN32
 #include <wchar.h> // _wgetenv for nvtx
 #endif
+#include <cuda_runtime.h>
 #include <nvToolsExt.h>
 #include <torch/csrc/utils/pybind.h>
 
 namespace torch::cuda::shared {
 
+struct RangeHandle {
```

Comment:
cc @ezyang, I see that you first added nvrtx bindings (7 years ago). do you want to take a look ?